tools/serverpod_cli/lib/src/database/create_definition.dart

```diff
@@ -83,6 +61,51 @@ DatabaseDefinition createDatabaseDefinitionFromEntities(
   );
 }
 
+bool _isViewTable(ClassDefinition classDefinition) {
+  if (classDefinition.viewName != null) {
+    return true;
+  }
+  return false;
+}
+
+String _getName(ClassDefinition classDefinition) {
+  if (classDefinition.viewName != null) {
+    return classDefinition.viewName!;
+  }
+  return classDefinition.tableName!;
+}
+
+List<IndexDefinition> _createIndexDefinition(ClassDefinition classDefinition) {
+  if (classDefinition.viewName != null) {
+    return [];
+  }
```

Comment:
Suggestion: add a validation rule that informs the user if an index is specified for a view.

Since we ignore any indexes specified for views (correctly) we should inform the developer that the view model is not correct.